gov-ithub/asistentul_cetateanului_frontend
# Use Auth0 Lock widget instead

#### Fixes issue #15 

#### Here are the changes I propose:
Use Lock widget instead of custom auth, custom auth doesn't allow custom fields for registration either way.

#### Test plan:
Open app:
  - Confirm view defaults to /home and you can't access /flux
  - Confirm you can sign in and get redirected to /flux
  - Confirm you can logout
  - Confirm you can sign up and get redirected to /flux
  - Confirm you can still see /home when signed in

#### Suggested reviewers: @gov-ithub/asistentul_cetateanului_frontend
